spec/features/artisan_questionnaire_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+feature "Artisan Questionnaire" do
+  let(:email) { "bernard.kanyolo+workdey1@andela.com" }
```

Comment:
Use Faker for models. Create a user then use the  user's email and password to login

    let(:user) { create(:user) 
    log_in_with(user.email, user.password)